west.yml

```diff
@@ -159,6 +159,11 @@ manifest:
       path: modules/hal/gigadevice
       groups:
         - hal
+    - name: hal_hpmicro
+      revision: pull/2/head
```

Comment:
Re-added the DNM label because this now points to a PR: https://github.com/zephyrproject-rtos/hal_hpmicro/pull/2